src/components/FilterBar.js

```diff
@@ -47,9 +53,9 @@ export default class FilterBar extends Component {
         options={options.transformedList}
         isLoading={options.isFetching}
         clearButton
-        defaultInputValue={defaultValue || ''}
```

Comment:
FYI I removed this because it conflicts with `selected`:

![image](https://user-images.githubusercontent.com/299129/36687058-4a4028ac-1add-11e8-98bf-bbd9c5843ecb.png)

There's a `defaultSelected` that can be used instead, but this caused other issues. It worked best by just passing in `path` directly into `selected` if it's set.